common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -2922,6 +2920,10 @@ def _copy_subdag(self, user_id, destination_version, block_key, source_blocks, d
                 raw=True,
                 block_defaults=new_block.defaults
             )
+            # _new_block doesn't copy the original_usage data needed for library blocks:
+            if new_block.edit_info.original_usage:
+                destination_block.edit_info.original_usage = new_block.edit_info.original_usage
+                destination_block.edit_info.original_usage_version = new_block.edit_info.original_usage_version
```

Comment:
@cpennington Do you really mean copying over *all* of the edit info, so that the "published" block and the "draft" block have the same edit_info? That would change the current behaviour whereby `previous_version` is set null when a block is first published, and `edited_on` is the time of publication.

i.e.
```python
destination_block = self._new_block(...)
destination_block.edit_info = EditInfo(new_block.edit_info.to_storable())  # or copy.deepcopy
```

Or do you mean the following, which is a slightly more general (and more opaque) version of what I already have:
```python
destination_block = self._new_block(...)
for key, val in new_block.edit_info.to_storable().iteritems():
    if getattr(destination_block.edit_info, key) is None:
        setattr(destination_block.edit_info, key, val)
```